package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "react_mp",
+  "version": "1.0.0",
+  "description": "ReactJS Mentoring Programm",
+  "main": "index.js",
+  "scripts": {
```

Comment:
When we create two different webpack configs as I have mentioned below, we will need to add `--config` param to the webpack command

Reply:
✔️ Done